ShitFood.Web/js/get-restaurants.js

```diff
@@ -0,0 +1,35 @@
+var axios = require('axios');
+
+function getRestaurants() {
+  var cutleryIcon = L.icon({
+    iconUrl: '../img/markers/cutlery.svg',
+    iconSize: [36, 36], // size of the icon
+    iconAnchor: [0, 0], // point of the icon which will correspond to marker's location
+    popupAnchor: [18, -3] // point from which the popup should open relative to the iconAnchor
+  });
+
+  axios.get('https://shitfoodapi.azurewebsites.net/api/getshit?lat=52.9548&lng=-1.1581')
+      .then(function (response) {
+        for (var i = 0; i < response.length; i++) {
```

Comment:
This needs a minor change as the `response` object isn't just the data you're expecting, it also has some other properties. So the DTO array is actually `response.data`, use that everywhere the the `.then`.

Reply:
Ah I didn't realise